features/taxation/applying_taxes/applying_correct_taxes_for_order_with_promotion_giving_discount_for_and_item_in_order_when_tax_rate_included_in_price.feature

```diff
@@ -0,0 +1,50 @@
+@applying_taxes
+Feature: Apply correct taxes on order item level when order has applied a promotion with discount for an item in order and tax rate is included in price
```

Comment:
Why? The title has 3 parts for all this file :
1. What we do (apply taxes for order)
2. what kind of discount this order has (because we have scenarios with 3 different promotios)
3. if the tax is included in price (because there are two kind of scenarios for every promotion)
If the title should be shorter then I propose merge scenarios included and excluded in price into one file, but before we  created scenarios for tax included in price in separate file (and for me it is correct, bacause `included in price` is not default for taxes).